resources/automate.rb

```diff
@@ -102,14 +101,11 @@
     notifies :reconfigure, 'chef_ingredient[automate]', :immediately
   end
 
-  if new_resource.enterprise.is_a?(String)
-    new_resource.enterprise = [new_resource.enterprise]
-    new_resource.enterprise.each do |ent|
-      execute "create enterprise #{ent}" do
-        command "automate-ctl create-enterprise #{ent} --ssh-pub-key-file=/etc/delivery/builder_key.pub > /etc/delivery/#{ent}.creds"
-        not_if "automate-ctl list-enterprises --ssh-pub-key-file=/etc/delivery/builder_key.pub | grep -w #{ent}"
-        only_if 'automate-ctl status'
-      end
+  Array(enterprise).each do |ent|
```

Comment:
`automate-ctl` will not allow you to create more than one enterprise on modern versions of Automate.